http/src/main/resources/reference.conf

```diff
@@ -103,6 +102,11 @@ monitor {
   pollPeriod = 15 seconds
   maxRetries = -1  # means retry forever
   recreateCluster = true
+  creatingTimeLimit = 1 hour
+  startingTimeLimit = 20 minutes
+  stoppingTimeLimit = 10 minutes
```

Comment:
Stopping a cluster can take a weirdly long time. I'd advocate for increasing this limit -- especially since most users don't explicitly stop but rely on auto-pause. Maybe 1 hour?

Maybe also 1 hour for deleting since that's not really user facing.

The other limits look good.